docs/sections/user_guide/yaml/components/filter_topo.rst

```diff
@@ -20,10 +20,18 @@ config:
 
 Configuration parameters for the ``orog_gsl`` component.
 
+  **filtered_orog:**
+
+  Name of the filtered output file.
+
   **input_grid_file:**
 
   Path to the tiled input grid file.
 
+  **input_raw_orog:**
+
+  Path to the raw orography file. The output of the ``orog`` driver.
```

Comment:
Is the `orog` driver the only place that can come from? The second sentence seems maybe a little over-specified given that filter topo was reading raw orog data long before `uwtools` existed. I'm not sure we have documentation yet that defines the inputs of components in terms of `uwtools` functionality and not just general requirements.

Reply:
Should I change this one to reference the output of the ufs_utils `orog` program? I do think that's the only thing that can create this file, independent of running with a UW driver.

Reply:
Yes, I think even just not referring directly to the UW driver would be good; referencing the `orog` _program_ would  be general enough IMO.